client/viame-web-common/components/Viewer.vue

```diff
@@ -382,8 +383,12 @@ export default defineComponent({
 <template>
   <v-main class="viewer">
     <v-app-bar app>
-      <navigation-title />
```

Comment:
Move NavigationTitle up to the ViewerLoader so that each platform can determine what to call itself.